src/styles/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  width: calc(200px - 2 * 16px - 2px);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container-image {
+  margin-top: 16px;
+  margin-bottom: 36px;
+  margin-left: 3px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__container-card-name {
+  margin-bottom: 4px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.card__container-product-code {
+  margin-bottom: 16px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary);
+}
+
+.card__reviews-number {
+  width: 53px;
+  font-weight: 300;
+  font-size: 10px;
+  line-height: 14px;
+  text-align: right;
+  color: var(--main-accent);
+}
+
+.card__container-price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__price {
+  width: 31px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary);
+}
+
+.card__price-number {
+  width: 50px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: right;
+  color: var(--main-accent);
+}
+
+.card__link {
+  text-decoration: none;
+}
+
+.card__buy-button {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-align: center;
+  padding: 12px 13px;
+  margin: auto;
```

Comment:
Pay attention to your CSS properties order
https://9elements.com/css-rule-order/